lib/diagnostic.rb

```diff
@@ -45,7 +57,16 @@
 # which returns the string "think think think".
 
 ##
-# your response here
+class Developer < Person
+
+  def initialize
+    @solve_problems = solve_problems
+  end
+
+  def solve_problems
+    puts "think think think"
```

Comment:
This will return `nil`, rather than the string, `'think think think'`, since `puts` only outputs to the console but doesn't have a return value.